packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -487,6 +510,10 @@ export const HelpCenterContactForm = () => {
 		return isSubmitting ? formTitles.buttonSubmittingLabel : formTitles.buttonLabel;
 	};
 
+	if ( mode === 'CHAT' && ! thirdPartyCookiesAllowed ) {
```

Comment:
To avoid showing the notice during loading:

```suggestion
	if ( mode === 'CHAT' && thirdPartyCookiesAllowed === false ) {
```

Reply:
Good call! https://github.com/Automattic/wp-calypso/pull/76777/commits/26b1c8e046893c2346e12b90c1075c37677a3a44